src/data/en/index.ts

```diff
@@ -2,6 +2,9 @@
 import { Instance } from '@/fetcher/graphql-types/operations'
 import { getLogoutUrl, getPasswordChangeUrl } from '@/helper/urls/get-auth-urls'
 import { headerData, footerData, landingSubjectsData,secondaryMenus } from './menu-data'
+
+const emailVerifiedSuccessfully = 'You have successfully verified your email address'
```

Comment:
It makes sense, but since we did not need this in the other 1000 lines I'd rather avoid it.
Maybe just `string.auth.messages.1080002` with a comment above?